<launch>

  <node pkg="learned_map" type="generateCRFFeatures" name="generateCRFFeatures" output="screen">

  <param name="loadMapName" value="/home/yh/mapModel/12.25/saveTestMap.vtk"/>
  <param name="loadTrajName" value="/home/yh/mapModel/12.25/icp_0301.txt"/>
  <param name="saveTxtName" value="/home/yh/test.txt"/>
  <param name="saveCloudName" value="/home/yh/test.vtk"/>
  <param name="mapLength" value="10560"/>
  <param name="kSearchMap" value="20" />
  <param name="kSearchTraj" value="600" />



  </node>
</launch>
